web/modules/custom/dpl_config_import/dpl_config_import.links.menu.yml

```diff
@@ -1,5 +1,5 @@
 config.sync:
-  title: 'Upload configuration'
-  description: 'Upload and import configuration from a YAML file'
+  title: "Upload configuration"
```

Comment:
All this quote changes wont be a problem in future config exports, or what?

Reply:
They should not.

The code style (and related changes) should not apply to exported configuration YAML - [only YAML in our modules and themes within the `web` folder](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/144/files#diff-03609cb60b0c6e92fb771eb8787d6722b8c31ca4c03eabc788e147acd8c6fb43R182).

Reply:
👍